alacritty/src/renderer/text/gles2.rs

```diff
@@ -444,17 +458,24 @@ pub struct TextShaderProgram {
 
     /// Rendering pass.
     ///
-    /// The rendering is split into 4 passes. One is used for the background and the rest to
-    /// perform subpixel text rendering according to
+    /// For dual source blending, there're 2 passes, one for background, another for text,
```

Comment:
I don't think it's gramatically incorrect, but "there're" is a bit awkward. So I'd suggest changing it:

```suggestion
    /// For dual source blending, there are 2 passes; one for background, another for text,
```